### ISSUE TITLE
rec:Observation inconsistent with sosa:Observation

### ISSUE THREAD

**Issue by dr-shorthair** [HajoRijgersberg/OM#23]
The conceptualization of `rec:Observation` in the RecordTable ontology is inconsistent with the W3C SSN Ontology. `rec:Observation rdfs:subClassOf prov:Entity .` while `sosa:Observation rdfs:subClassOf prov:Activity .` - see https://www.w3.org/TR/vocab-ssn/#PROV_Alignment

**Comment by HajoRijgersberg** [comment]
Hi Simon,
I have forwarded your issue to the author of RecordTable. Hope to hear from him soon!
Best, Hajo

**Comment by jantop** [comment]
You are right in the sense of 'making an observation' can best be modelled as a prov:Activity. However,  in our context we refer to the 'description of an observation', which is best modelled as a prov:Entity. 
But the issue has been solved by the new version of RDF Record Table, which we urgently need to publish as well. This version does not use the concept  rec:Observation anymore. This is the paper describing this version. 
[intsys_v8_n12_2015_11(1).pdf](https://github.com/HajoRijgersberg/OM/files/3824751/intsys_v8_n12_2015_11.1.pdf)

**Comment by dr-shorthair** [comment]
If your intention is to classify the 'description of an observation' then perhaps the class should be named `rec:ObservationRecord`.

**Comment by jantop** [comment]
Good suggestion. In the new version of RDF Record Table we define the notion rec:RecordTable, so in a way it follows your proposal. It represents both an 'observation record' (a - possibly nested -  row in a table) and the entire  table.
